client/my-sites/sidebar/sidebar.jsx

```diff
@@ -345,13 +346,25 @@ export class MySitesSidebar extends Component {
 	};
 
 	store() {
-		const { canUserManageOptions, isJetpack, site, siteSuffix, translate } = this.props;
+		// IMPORTANT: If you add a country to this list, you must also add it
+		// to ../../extensions/woocommerce/lib/countries in the getCountries function
+		const allowedCountryCodes = [ 'US', 'CA' ];
+		const { currentUser, canUserManageOptions, isJetpack, site, siteSuffix, translate } = this.props;
 		const storeLink = '/store' + siteSuffix;
 		const showStoreLink = config.isEnabled( 'woocommerce/extension-dashboard' ) &&
 			site && isJetpack && canUserManageOptions && this.props.isSiteAutomatedTransfer;
 
+		if ( ! showStoreLink ) {
```

Comment:
Since the country filtering is temporary, I think I'd like to just keep it the way it is for now. The country filtering likely won't last more than 6 months anyway.